proteus/mprans/SedClosure.h

```diff
@@ -576,7 +576,7 @@ namespace proteus
 
     
 
-    inline double*  mIntFluid(  double sedF,
+    inline void  mIntFluid( double * mint2, double sedF,
```

Comment:
On functions like this that "return" values by setting function arguments, there's an informal (and not consistently followed) convention to put the  inputs first and the outputs last.  mint2 is the output here, right?